app/partials/main-header.cjsx

```diff
@@ -56,6 +57,7 @@ module.exports = React.createClass
           <Link to="projects" className="main-nav-item"><Translate content="mainNav.projects" /></Link>
           <Link to="about" className="main-nav-item"><Translate content="mainNav.about" /></Link>
           <Link to="talk" className="main-nav-item"><Translate content="mainNav.talk" /></Link>
+          <Link to="notifications" className="main-nav-item"><Translate content="mainNav.notifications" /></Link>
```

Comment:
This links to notifications from all projects/sections.

I imagine we'd eventually integrate this into the personalized sign in page, but until we get to it I'm guessing it's a top level link.